twoliter/embedded/Dockerfile

```diff
@@ -179,7 +214,7 @@ RUN --mount=target=/host \
         --downloadonly \
         --downloaddir . \
         --forcearch "${ARCH}" \
-        install $(printf "bottlerocket-%s\n" ${PACKAGES}) \
+        install $(printf "bottlerocket-%s\n" metadata ${PACKAGES}) \
```

Comment:
`printf` is prefixing everything here with `bottlerocket-`, so the package is actually called `bottlerocket-metadata`.